nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/shape/OneHot.java

```diff
@@ -76,8 +76,21 @@ public OneHot(INDArray indices, INDArray output, int depth, int axis, double on,
         addArgs();
     }
 
+    //TODO: Make sure this is correct.
+    public OneHot(INDArray indices, int depth, int axis, double on, double off) {
+        this(indices, null, depth, axis, on, off);
+    }
 
+    //TODO: Make sure this is correct.
+    public OneHot(INDArray indices, int depth, int axis, double on, double off, DataType dataType){
+        this(indices, null, depth, axis, on, off);
+        this.outputType = dataType;
+        addArgs();
```

Comment:
remove this addArgs call in this constructor (it's already called in the constructor called by `this(...)`).
Otherwise LGTM